quipucords/scanner/vcenter/inspect.py

```diff
@@ -71,6 +73,18 @@ def run(self):
                 self.connect_scan_task.id)
             return ScanTask.FAILED
 
+        # Prep inspect result
```

Comment:
Probably should make this method atomic since it is doing several db transactions

Reply:
done